service/templates/base.html

```diff
@@ -71,7 +71,7 @@
                     <span>Skip to main content</span>
                 </a>
                 <a class="logo" href="//jisc.ac.uk">
-                    <img alt="Jisc logo" src="{{url_for('static', filename='images/jisc-logo.png')}}">
+                     <img alt="Jisc logo" width="43" height="25" src="{{url_for('static', filename='images/jisc-logo.png')}}">
```

Comment:
Makes sense to use just one image to cut down requests, and scale depending on where it needs to sit. Does it make sense to use an SVG (for better scaling and smaller file sizes) (or does that complicate matters)?

@tharglet Re: where to define width, colleagues are telling me CSS (whether png or css) is better.